### ISSUE TITLE
status code 500 with no database connection

### ISSUE THREAD

**Issue by wp-coin** [lando/lando#3715]
docker  24.0.0 ( downgraded)
ubuntu 22 wsl2

Lando v3.21.0-beta.18

I am getting an error  (Request failed with status code 500)

**Comment by AaronFeledy** [comment]
This error is likely not related to Docker. Have you tried running `lando rebuild`? In order to help, we need more information about your setup, what you're trying to accomplish, what steps lead to the error, what troubleshooting steps you have tried, etc.

**Comment by AaronFeledy** [comment]
I think that's just how wordpress works. If the database isn't connected, it 301 redirects you to setup. If the lndo.site domain is working but localhost is not, it's likely that your Wordpress is configured in a way that 302 redirects you away from the localhost domain. If that's the case, these aren't errors, but Wordpress trying to direct you to the correct urls.

**Comment by joshuami** [comment]
It may be related that after running `lando update`, I'm no longer getting an ARM compatible database image. For me at least, that seems to be the cause of the 500 errors.

**Comment by joshuami** [comment]
Sorry, that likely has nothing to do with your issue seeing that you are on WSL. My bad. I didn't read carefully enough. 😉

**Comment by reynoldsalec** [comment]
@wp-coin the 300's are likely an initial response code that redirects to a 200; the Lando scanner doesn't follow redirects, so 300's would be expected. See the scanner docs for more details and options to configure: https://docs.lando.dev/core/v3/scanner.html

Is your database is connecting correctly now?

**Comment by AaronFeledy** [comment]
Lando indeed checks the status of the individual services. Only after it confirms that they're up does it start the scanner. The purpose of the scanner is to get a status from your application to see if it's functioning in the Lando environment. A good indication of a functioning application is a 200 or 3xx response.

**Comment by AaronFeledy** [comment]
Lando leaves it entirely up to you to make your application work the way you like. If your app's code checks the domain name and issues a redirect based on that then Lando will stay out of the way.

It seems you were able to get your database connected and we've verified that Lando is not preventing your application from issuing its desired redirects so I'm going to close this issue. Feel free to open another issue or ask for support in the [Slack Channel](https://launchpass.com/devwithlando) if you need anything else.
